app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchPeopleViewModel.kt

```diff
@@ -275,7 +269,7 @@ abstract class SearchPeopleViewModel(
         }
     }
 
-    abstract suspend fun getInitialContacts(): SearchResult
+    abstract fun getInitialContacts(): Flow<SearchResult>
```

Comment:
Nice catch, good topic for discussion on the AC. I'll prepare a couple a detekt rules for this if we agree on this topic.